packages/internal-test-helpers/lib/node-query.ts

```diff
@@ -26,23 +37,29 @@ export default class NodeQuery {
     Object.freeze(this);
   }
 
-  find(selector) {
+  find(selector: string | Element) {
     assertSingle(this);
 
     return this[0].querySelector(selector);
   }
 
-  findAll(selector) {
-    let nodes = [];
+  findAll(selector: string) {
+    let nodes: HTMLElement[] = [];
 
     this.nodes.forEach((node) => {
-      nodes.push(...node.querySelectorAll(selector));
+      nodes.push(...(node.querySelectorAll(selector) as NodeListOf<HTMLElement>));
```

Comment:
If so, then this can also drop the cast.

```suggestion
      nodes.push(...node.querySelectorAll(selector));
```